src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DocumentPullDiagnosticsHandler.cs

```diff
@@ -79,15 +79,17 @@ public async Task<DiagnosticReport[]> HandleRequestAsync(DocumentDiagnosticsPara
                 throw new ArgumentNullException(nameof(clientCapabilities));
             }
 
+            _logger.LogInformation($"Starting request for {request.TextDocument.Uri}.");
```

Comment:
> I'm hesitant to have this granularity of logging just because the logs could become unruly to evaluate. I know that @CyrusNajmabadi built a system for structured logging to capture scopes effectively, is that something we can reuse in these handlers?


I see a trace start/stop, but I don't think it would have any effect on the log output size:
https://github.com/dotnet/roslyn/pull/50416/files#diff-88905c68cabc4c9fb4a13d39f2960375a4d2527f1ed2f89434bc93ce53d6b926R81
https://github.com/dotnet/roslyn/pull/50416/files#diff-feb8029df6857f259c39b5dac957b3e5c1183f9fa9bd8f062aa54cf0f9023bf4R48

Reply:
> Also for pull diagnostics given it's polled every few seconds I imagine these logs will be the vast majority of all logs in the output which is concerning. @CyrusNajmabadi do you all log every pull diagnostic request you receive regardless of "no-op" ability?

It appears so:
https://github.com/dotnet/roslyn/pull/50416/files#diff-746af321ae8dde2dee926532a1b6ff5e20168af0dcfd2a50b86bb97d4cba28c6R110